WixInstaller/branding.wxs

```diff
@@ -34,44 +34,6 @@
 
 <Wix xmlns="http://schemas.microsoft.com/wix/2006/wi" xmlns:netfx="http://schemas.microsoft.com/wix/NetFxExtension">
     <?include branding.wxi ?>
```

Comment:
This file does not need to change since it is overwritten by the internal.

Reply:
I forgot: we do need checks in the code to only use the healthcheck for Stockholm and earlier hosts.

Reply:
In my understanding, Stockholm and earlier builds won't trigger the copy of the file from xenadmin-branding repo, therefore I just don't need to touch this file in xenadmin repo. Is it right?